session/session.go

```diff
@@ -1418,7 +1418,7 @@ func logStmt(node ast.StmtNode, vars *variable.SessionVars) {
 
 func logQuery(query string, vars *variable.SessionVars) {
 	if atomic.LoadUint32(&variable.ProcessGeneralLog) != 0 && !vars.InRestrictedSQL {
-		query = executor.QueryReplacer.Replace(query)
+		query = executor.QueryReplacer.Replace(query) + vars.GetExecuteArgumentsInfo()
```

Comment:
I think put `vars.GetExecuteArgumentsInfo()` to line 1422 is better. Reduce the processing of strings.